public/js/roster.js

```diff
@@ -60,9 +60,13 @@ function createTable() {
                                 ${ row.name }
                             </a>
                         </li>
-
-                        ${ row.raid_name || row.class ? `
+                        ${ row.is_alt || row.raid_name || row.class ? `
                             <li>
+                                ${row.is_alt ? `
+                                <span class="font-weight-bold">
+                                    <span class="role-circle" style="color:orange">Alt</span>
+                                </span>
+                                ` : ''}

```

Comment:
You will also want to add an indicator on the item list page: https://github.com/thatsmybis/thatsmybis/blob/master/public/js/itemList.js#L165